src/components/Footer.jsx

```diff
@@ -0,0 +1,94 @@
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import { Container } from "react-bootstrap";
```

Comment:
Is bootstrap geriau importink po viena module koki naudoji per default import. 
![image](https://github.com/front-end-by-rimantas/46-grupe-teamwork-martex/assets/135387482/11d5eb4b-5f5d-4d8b-8e35-e82683cd0ab3)